lib/ansible/modules/cloud/ovirt/ovirt_templates.py

```diff
@@ -246,6 +297,86 @@ def post_import_action(self, entity):
         self._service = self._connection.system_service().templates_service()
 
 
+def _get_role_mappings(module):
+    if module.params['role_mappings'] is None:
```

Comment:
Since the default is `[]` I think this can't be none never

Reply:
will remove